core/templates/dev/head/domain/question/EditableQuestionBackendApiService.ts

```diff
@@ -122,14 +122,17 @@ angular.module('oppia').factory('EditableQuestionBackendApiService', [
       });
     };
 
-    var _deleteQuestionFromSkill = function(
```

Comment:
Don't we need this function anymore?

Reply:
Ah, this is due to the way github checked the diff. I removed the function above this `_addQuestionSkillLink`, since that is never used and added `_changeDifficulty`, but since I replaced it in reverse order, it is shown as if this is deleted. 
This function still exists above :).

Reply:
Hmmm, my bad. I didn't notice this function before!